### PR TITLE
Odysee | Atomic - Periodic Table | Monochromatic | Cams

### PR DESCRIPTION
**Pull request by shuvashish76**
Odysee: already available on F-droid app.
Atomic - Periodic Table - https://github.com/JLindemann42/Atomic-Periodic-Table.Android/issues/19
Monochromatic - project abandoned but really useful app. One of its own kind app, I request you to keep it here.
Cams - no issue, available on F-Droid

---

Update ToC Image Manipulation (#222)

---

Topic of discussion: 

[Marcel Bokhorst](https://github.com/M66B) archived all his projects & closed related XDA threads, which includes popular apps e.g. [XPrivacy](https://github.com/M66B/XPrivacy), [FairEmail](https://github.com/M66B/FairEmail), [NetGuard](https://github.com/M66B/NetGuard).

https://forum.xda-developers.com/t/closed-app-5-0-fairemail-fully-featured-open-source-privacy-oriented-email-app.3824168/page-1087

Short version: https://github.com/MuntashirAkon/AppManager/discussions/754

But lets keep them here & see how it goes

### CODE REVIEW

**Review by offa — APPROVED** [review]

**Comment by offa** [comment]
> Marcel Bokhorst archived all his projects & closed related XDA threads, which includes popular apps e.g. XPrivacy, FairEmail, NetGuard.

Thanks, really bad news :-(

> But lets keep them here & see how it goes

Indeed, hopefully the projects continue somehow. :crossed_fingers:

**Comment by offa** [comment]
> Update ToC Image Manipulation (#222)

Done, thanks!
